server/src/main/java/org/apache/iotdb/db/engine/compaction/level/LevelCompactionTsFileManagement.java

```diff
@@ -191,13 +191,13 @@ private void deleteLevelFile(TsFileResource seqFile) {
       List<TsFileResource> result = new ArrayList<>();
       if (sequence) {
         List<SortedSet<TsFileResource>> sequenceTsFileList =
-            sequenceTsFileResources.get(timePartition);
+            sequenceTsFileResources.getOrDefault(timePartition, new ArrayList<>());
```

Comment:
Should we use computeIfAbsent to set the ArrayList back?

Reply:
ok, computeIfAbsent is a better choice

Reply:
It seem that the tests failed if change here…

Reply:
this only for a check, he will add the tsfile in another place